frontend/packages/@depmap/dataset-manager/src/components/index.tsx

```diff
@@ -47,8 +45,20 @@ export default function Datasets() {
     [key: string]: string;
   } | null>(null);
 
+  // TODO: Remove
```

Comment:
Very minor, but it'd be good to include the reason why like you did elsewhere (ie: "The endpoints for feature type and sample type are deprecated and should not be used.")

Seeing a TODO like "Remove" leaves me wondering why.